.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -43,3 +43,29 @@ Relevant software versions:
 ## Additional context
 
 <!-- Add any other context about the problem here, such as stack traces or debugging info. -->
+
+#### Definition of Done
+
+- [ ] This bug is **done**:
+  - [ ] Bug resolved to **user's** satisfaction
+  - [ ] Automated tests are passing
+  - [ ] Code is peer reviewed and pushed to master
+  - [ ] Deployed successfully to test environment
+  - [ ] Checked against CODING guidelines
+  - [ ] Relevant new tasks are added to backlog and communicated to the team
+  - [ ] Important decisions recorded in the issue ticket
+  - [ ] Sprint board is updated
```

Comment:
I don't understand how exactly "Sprint board should be updated"

Reply:
Means it moved to the right column in the Zenhub Geostore board.  But maybe that is too much detail? Or not so useful?